be/src/runtime/tablets_channel.cpp

```diff
@@ -234,6 +250,7 @@ Status TabletsChannel::_open_all_writers(const PTabletWriterOpenRequest& request
         wrequest.tuple_desc = _tuple_desc;
         wrequest.slots = index_slots;
```

Comment:
_dicts is constructed from slots, but wrequest already has a field slots so that I think the node could parse dict from wrequest's slots field. No need to create dicts here.